html/template.html

```diff
@@ -71,7 +71,9 @@
 <?php if (isset($config['display_testnet']) && $config['display_testnet']) { ?>
       <p class="stat"><span class="label">Testnet:</span> <?php echo ($data['testnet']) ? "Yes" : "No"; ?></p>
 <?php } ?>	 
-      <p class="stat"><span class="label">Node IP Address:</span> <?php echo $data['node_ip']; ?></p>
+<?php if(isset($config['display_ip']) && $config['display_ip']) { ?>
```

Comment:
Ok i have done it, but there seems to be an other problem then the spaces ?.. :S